src/diem/offchain/command.py

```diff
@@ -2,18 +2,20 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from abc import ABC, abstractmethod
+
+from . import CommandRequestObject
 from .action import Action
 
 import typing
 
 
 class Command(ABC):
     @abstractmethod
-    def id(self) -> str:
+    def command_type(self) -> str:
         ...
 
     @abstractmethod
-    def is_inbound(self) -> bool:
```

Comment:
I think it is necessary to mark a command is inbound.

It is always required information when processing the command.